src/main/flight/mixer.c

```diff
@@ -662,7 +663,7 @@ static void applyFlipOverAfterCrashModeToMotors(void)
 
         // Apply a reasonable amount of stick deadband
         const float flipStickRange = 1.0f - CRASH_FLIP_STICK_MINF;
-        float flipPower = MAX(0.0f, stickDeflectionLength - CRASH_FLIP_STICK_MINF) / flipStickRange;
+        float flipPower = MAX(0.0f, stickDeflectionLength - CRASH_FLIP_STICK_MINF) / (flipStickRange / ((float)mixerConfig()->crashflip_power_percent / 100.0f));
```

Comment:
The cast to `float` and the extra parenthesis are unnecessary. The logic would be clearer as:
```
        const float flipPowerFactor = mixerConfig()->crashflip_power_percent / 100.0f;
        float flipPower = MAX(0.0f, stickDeflectionLength - CRASH_FLIP_STICK_MINF) / flipStickRange * flipPowerFactor;
```

Reply:
Is the cast to float unnecessary because crashflip_power_percent will implicitly be converted to a float?
Why create a second variable flipPowerFactor? Wouldn't this take up slightly more memory?

Thanks for the feedback, I'm always trying to learn :)

Reply:
> Is the cast to float unnecessary because crashflip_power_percent will implicitly be converted to a float?

Yes, since it's in a math statement with other floats.

> Why create a second variable flipPowerFactor? Wouldn't this take up slightly more memory?

No. The compiler will optimize it to be equivalent. Plus with `flipPowerFactor` being a `const` it will further help this to occur. Splitting the formula apart like this makes it much easier to understand what's going on.